specs/kyber/src/sampling.rs

```diff
@@ -149,8 +150,7 @@ fn sum_coins(coins: &mut BitVectorChunks<'_>) -> KyberFieldElement {
 pub fn sample_poly_cbd(eta: usize, bytes: &[u8]) -> KyberPolynomialRingElement {
```

Comment:
We don't need an iterator here really, but could just use a slice as in the spec (`&bits[2 * i * eta..2 * i * eta + eta]` etc.)

Reply:
I did `bits.next().unwrap()`instead since I thought that'd be easier to read.